wcomponents-theme/src/main/js/wc/fix/inlineBlock_ie8.js

```diff
@@ -38,16 +38,6 @@ define(["wc/has", "wc/dom/classList", "wc/timers"], function(has, classList, tim
 		}
 	}
 
-	if (has("ie") === 8) {
```

Comment:
This is the right spirit however I suggest we delete the entire JS file (and remove any calls to it elsewhere).

Reply:
I concur